examples/basic/lines/arcs.py

```diff
@@ -1,3 +1,14 @@
+''' A arc graph using pre-defined data points. This example
+demonstrates the use of the ``arc`` method to make a graph
+by drawing three arcs of defined radius, defined start and
+end angles at defined point.
```

Comment:
```suggestion
by drawing three arcs of defined radius, start and
end angles at a specified point.
```

In my opinion the word "defiend" is redundant and could appear only once.